mail/notmuch/Portfile

```diff
@@ -41,7 +40,6 @@ depends_lib         port:gmime3 \
                     port:zlib
 
 patchfiles          ${distname}-configure.patch \
-                    ${distname}-doc-makefile.patch \
```

Comment:
Please also remove the patch file from git.

Reply:
Yeah, about that: I was not sure what would happen if somebody wanted to install version 0.29 instead of 0.29.1 (for whatever reason). For Gentoo Linux, we only remove patches tied to a particular version when the ebuild (i.e. the Gentoo pendant to a MacPorts port) is also removed.

I can remove the patch, just did not want to foul things up.

Reply:
I'm not familiar with Gentoo's packaging, but MacPorts typically only offers a single version of a port, such that when a port is updated, the previous version is no longer offered. (Reverting to an old version of a port takes more effort as a result. If multiple versions of the software are offered, it is normally done using separate ports.)

Reply:
Oddly enough, I did not realise that MacPorts basically only offers a single version for each port. Gentoo distinguishes between stable and unstable builds (the latter being less extensively tested and usually not problematic in any way), and that for different supported processor architectures. It is even normal to have more than one unstable build per architecture, to enable users to fall back if the latest version of packaged software XYZ introduces errors.

In any case, I removed the patches for version 0.29. 😉